detox/__tests__/setupJest.js

```diff
@@ -1,94 +1,3 @@
 jest.mock('proper-lockfile');
 jest.mock('signal-exit');
 jest.mock('../src/logger/DetoxLogger');
-
-const path = require('path');
-
-const _ = require('lodash');
-const yargs = require('yargs');
-
-function callCli(modulePath, cmd) {
```

Comment:
I tried to understand why I had been using `global.callCli` instead of importing it from a helper file, and to be frank, I have no idea. Therefore, I moved it to `detox/__tests__/helpers.js` as an exported function, and it worked flawlessly in the unit tests.

That was a huge inspiration to repeat this success for the other `global.*` helpers you introduced in this PR.